setup.py

```diff
@@ -24,13 +40,8 @@ def read(fname):
     packages=find_packages(exclude=("test*", )),
     package_dir={'marshmallow-jsonschema': 'marshmallow-jsonschema'},
     include_package_data=True,
-    install_requires=['marshmallow>=2.9.0'],
-    tests_require=[
-        'coverage>=4.5.3',
-        'jsonschema>=2.6.0',
-        'pytest>=4.6.3',
-        'strict-rfc3339',
```

Comment:
I haven't added this packages to `requirements-test.txt` because I'm not quite sure what it brings to test suite as it's not imported there. Could you please tell what it's used for?